inc/form.class.php

```diff
@@ -1353,21 +1357,20 @@ public function duplicate() {
                }
 
                $new_target_change = new PluginFormcreatorTargetChange();
-               $new_target_change->add($update_target_change);
-               $new_target_item_id = $new_target_change->getID();
-               if (!$new_target_item_id) {
+               $update_target_change['title'] = $update_target_change['name'];
+               if (!$new_target_change->update($update_target_change)) {
                   return false;
                }
                break;
          }
 
-         $target->update([
-            'id'        => $target->getID(),
-            'items_id'  => $new_target_item_id,
-         ]);
-
```

Comment:
this update does nothing becasue $target is already the duplicated instance. No need to update the items_id column